design-at-cornell/src/homepage/Homepage.tsx

```diff
@@ -1,106 +1,91 @@
-            <SubComponentTitle>Community</SubComponentTitle>
-            <p>Short Description about the design community to intrigue and interest users</p>
-            <ExploreSubtitle> Explore Community {'>'}</ExploreSubtitle>
-          </SubComponentVerticalFlex>
-        </SubComponent>
-      </SubComponentHorizontalFlex>
-    </SubComponentVerticalFlex>
+    <MissionStat>
+      Our mission is to connect, inform, and inspire the design community at Cornell University.
+    </MissionStat>
+    <Grid>
+      <Col>
+        <ElementContainer>
+          <Link to="./explore-courses">
+            <img src={design_courses} alt={'Design Courses'} />
+          </Link>
+          <CardTitle>Design Courses</CardTitle>
+          <p>A central hub to learn about courses to enrich your design skillset</p>
+          <ExploreSubtitle to="./explore-courses"> Explore Courses {'>'}</ExploreSubtitle>
+        </ElementContainer>
+        <ElementContainer>
+          <Link to="./events">
+            <img src={events_icon} alt={'Events'} />
+          </Link>
+          <CardTitle>Events</CardTitle>
+          <p>Find events to learn, grow, and share with your creative community</p>
+          <ExploreSubtitle to="./events"> Explore Events {'>'}</ExploreSubtitle>
+        </ElementContainer>
+      </Col>
+      <Col>
+        <ElementContainer>
+          <Link to="./areas-of-study">
+            <img src={majors_minors} alt={'Majors/Minors'} />
+          </Link>
+          <CardTitle>Majors {'&'} Minors</CardTitle>
+          <p>A guide to the different paths you may pave at Cornell</p>
+          <ExploreSubtitle to="./areas-of-study"> Explore Majors & Minors {'>'}</ExploreSubtitle>
+        </ElementContainer>
+        <ElementContainer>
+          <Link to="./articles">
+            <img src={resources_icon} alt={'Articles'} />
+          </Link>
+          <CardTitle>Articles</CardTitle>
+          <p>Increase your knowledge of design through relevant articles</p>
+          <ExploreSubtitle to="./articles"> Explore Articles {'>'}</ExploreSubtitle>
+        </ElementContainer>
+      </Col>
+      <Col>
+        <ElementContainer>
+          <Link to="./student-organizations">
+            <img src={orgs_icon} alt={'Student Organizations'} />
+          </Link>
+          <CardTitle>Organizations</CardTitle>
+          <p>A place to explore your passion through design organizations at Cornell</p>
+          <ExploreSubtitle to="./student-organizations">
+            {' '}
+            Explore Organizations {'>'}
+          </ExploreSubtitle>
+        </ElementContainer>
+        <ElementContainer>
+          <Link to="./design-community">
+            <img src={community_icon} alt={'Faculties'} />
+          </Link>
+          <CardTitle>Faculty</CardTitle>
+          <p>Explore and connect with the design faculty here at Cornell</p>
+          <ExploreSubtitle to="./design-community"> Explore Community {'>'}</ExploreSubtitle>
```

Comment:
I think it might make more sense for the Faculty section to have a link that says "Explore Faculty" rather than "Explore Community".